operator/pkg/ingress/ingress.go

```diff
@@ -610,8 +610,8 @@ func (ic *Controller) regenerate(ing *slim_networkingv1.Ingress, forceShared boo
 		"model":        m,
 	}).Debug("Generated model for ingress")
 	cec, svc, ep, err := translator.Translate(m)
-	// Propagate Ingress annotation if required. This is applicable only for dedicated LB mode.
-	// For shared LB mode, the service annotation is defined in other higher level (e.g. helm).
+	// Propagate Ingress annotation and label if required. This is applicable only for dedicated LB mode.
+	// For shared LB mode, the service annotation and label are defined in other higher level (e.g. helm).
```

Comment:
```suggestion
	// Propagate Ingress annotations and labels if required. This is applicable only for dedicated LB mode.
	// For shared LB mode, the service annotations and labels are defined in other higher level (e.g. helm).
```

Reply:
How about copying same annotations with set prefixes as labels to LB service as well? Would that be acceptable? 
OR should we essentially duplicate the exact annotation logic for labels?

Reply:
I think duplicating the logic is better; annotations and labels are technically distinct, even though a lot of folks use them interchangeably.

Reply:
We should reuse the prefix config setting for both annotations and labels though.

Reply:
@youngnick is it too much of a duplicate now?

Reply:
hmm, yes, it doesn't feel like we'll need separate configurability for labels vs annotations to me. I'd probably rather see a single labels+annotations prefixes config setting that applies to both.

Reply:
Just so I understand, do you mean the user facing config to be same parameter that applies to both annotation and label?

If so, do you think we can run into a case where user expects to have different prefixes set for both?

Reply:
I think that it's unlikely that the user will want that, and if people do ask for it, we can bring it back if necessary. But having separate config feels like YAGNI (You Ain't Gonna Need It) to me.